sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts

```diff
@@ -757,12 +762,16 @@ export class ConfirmationOfEnrollmentService {
   /**
    * Generates the next document number to be associated
    * with a disbursement.
+   * @param offeringIntensity offering intensity.
    * @returns sequence number for disbursement document number.
    */
-  private async getNextDocumentNumber(): Promise<number> {
+  private async getNextDocumentNumber(
+    offeringIntensity: OfferingIntensity,
+  ): Promise<number> {
+    const sequenceGroupName = `${offeringIntensity}_DISBURSEMENT_DOCUMENT_NUMBER`;
```

Comment:
I did not follow the reason why the const was removed instead of just used in the concatenation.
```ts
const sequenceGroupName = `${offeringIntensity}_${DISBURSEMENT_DOCUMENT_NUMBER_SEQUENCE_GROUP}`.
```
Can you please clarify the indentation just for my understanding?